src/rollback.cpp

```diff
@@ -495,13 +495,10 @@ const std::list<ActionRow> RollbackManager::actionRowsFromSelect(sqlite3_stmt* s
 			row.nodeMeta    = sqlite3_column_int(stmt, 8);
 		}
 
-		if (row.type == RollbackAction::TYPE_SET_NODE || row.nodeMeta) {
+		if (row.type == RollbackAction::TYPE_SET_NODE) {
```

Comment:
it's apparent that this part is meant to apply for both `TYPE_MODIFY_INVENTORY_STACK` and `TYPE_SET_NODE`, so this change is just plain incorrect